src/components/swap/SwapBuyFiatButton.tsx

```diff
@@ -130,7 +137,12 @@ export default function SwapBuyFiatButton() {
         element={InterfaceElementName.FIAT_ON_RAMP_BUY_BUTTON}
         properties={{ account_connected: !!account }}
       >
-        <StyledTextButton onClick={handleBuyCrypto} disabled={buyCryptoButtonDisabled} data-testid="buy-fiat-button">
+        <StyledTextButton
+          onClick={handleBuyCrypto}
+          disabled={buyCryptoButtonDisabled}
+          data-testid="buy-fiat-button"
+          shouldFadeIn={shouldFadeIn.current}
```

Comment:
won't the fade in only occur if the opacity was not initially 1.0? in this case, is it necessary for shouldFadeIn to be used? Couldn't it just always be applied?

Reply:
I answered this above, but shouldFadeIn is for when the country is not in cache and it will fade in later. however, if it is cached then it will never fade in and the opacity will always be set to 1.0